components/avatars/avatar.css

```diff
@@ -0,0 +1,23 @@
+.apex-avatar{
+    margin: 5px;
+    border-radius: 50%;
+}
+.avatar-round-sm{
+    width: 2rem;
+    height: 2rem;
+}
+.avatar-round-md{
+    width: 4rem;
+    height: 4rem;
+}
+.avatar-round-lg{
+    width: 6rem;
```

Comment:
If possible use var here, that will help you in future

Reply:
Yes. I'll keep this is mind.
Thanks for the suggestion.